app/players/dwarves/dwarf_warrior.py

```diff
@@ -0,0 +1,20 @@
+from app.players.dwarves.dwarf import Dwarf
+
+
+class DwarfWarrior(Dwarf):
+
+    def __init__(self,

```

Comment:
After opening the parentheses, write all the arguments on a new line, including the first one